src/HttpKernel/ContaoCache.php

```diff
@@ -37,7 +37,9 @@ public function __construct(HttpKernelInterface $kernel, $cacheDir = null)
     {
         parent::__construct($kernel, $cacheDir);
 
-        $this->addSubscriber(new HeaderReplaySubscriber());
+        $this->addSubscriber(new HeaderReplaySubscriber(
+            ['ignore_cookies' => '/^csrf_.+/']
```

Comment:
I think this should be:

```php
['ignore_cookies' => ['/^csrf_/']]
```

Reply:
I don't think that's correct no. It should only match cookies starting with `csrf_` and a following key. It should not match `csrf_` standalone.

Reply:
That would be `/^csrf_./` then :)

But more importantly it should be an array:  
`'ignore_cookies' => ['/^csrf_./']` 
instead of 
`'ignore_cookies' => '/^csrf_./'`

Reply:
> That would be /^csrf_./ then :)

No? That would only match `csrf_f` but not `csrf_foobar`? Options fixed in d3c5768e3d0791e3660eacf3327e19b0f94bf979.

Reply:
`/^csrf_./` would match `csrf_foobar` because there is no `$` at the end of the regex. But that’s a very minor optimization so I think you can keep it as it is now. 🙃